README.md

```diff
@@ -103,13 +103,19 @@ where
 (this is basically the output of the MLC method)
 - ``true_labels`` is a comma-separated list of true labels (ground truth)
 
-Note that you can omit the ``true_labels`` section if you only want to predict the 
+Note that you can omit the ``true_labels`` section if you only want to predict the
 quality of the MLC result.
 
 For example, a row in the data file could look like this:
 
 ``Optimal investment policy of the regulated firm\tConcept0:0.5,Concept1:1\tConcept0,Concept3``
 
+You have the option to provide a separate data split for Label Calibration training
+(see section Train for more information). In this case,
```

Comment:
Why uppercase for "Section" ?

Reply:
Because that is how you reference parts of a text, I think? "see Chapter 5, see Subsection 6.1" https://ellii.com/blog/the-capitalization-of-certain-terms-exercise-page-chapter-unit-paragraph-line